src/cli/submit/data-set/DataSet.definition.ts

```diff
@@ -27,14 +27,43 @@ export const SubmitDataSetDefinition: ICommandDefinition = {
             description: "Submit a job from the data set \"ibmuser.cntl(iefbr14)\" and print only the job ID",
             options: "\"ibmuser.cntl(iefbr14)\" --rff jobid --rft string"
         },
+        {
+            description: "Submit a job from the data set \"ibmuser.cntl(iefbr14)\" and wait for job complete.",
+            options: "\"ibmuser.cntl(iefbr14)\" --wait 5,12"
+        },
     ],
     positionals: [{
         name: "dataSet",
         description: "The data set containing JCL that you would like to submit",
         type: "string",
         required: true
     }],
-    options: [],
+    options: [
+        {
+            name: "wait",
+            aliases: ["w"],
+            description: "Specify job query interval and max times of querying job status. " +
+            "The format of this option is comma-separated numeric values. For example, '5,12' means queries job status every 5 seconds for 12 times at most.",
+            type: "string",
+            required: false,
+            conflictsWith: ["wait-for-active", "wait-for-output"]
```

Comment:
I do like that we are making full use of the imperative framework here.
Based on how Zowe CLI has handled end-user confusion with conflicting options, should there be separate objects ? 
i.e. `zowe zftp submit job-wait-for-output` and `zowe zftp submit  job-wait-for-active`
@dkelosky @ChrisBoehmCA @tiantn ?

Reply:
Currently these three options(--wait, --wait-for-output and --wait-for-active) are independent of each other, if you write one you cannot write the other two, otherwise you will get an error. These are consistent with zowe CLI with z/OSMF profile. Do you mean let them as new objects to better understand for end-user? 😊

Reply:
@zFernand0  is using a bit of Zowe CLI jargon when he says "objects". He means splitting them into two different commands. I would disagree with that suggestion personally, to be consistent with the base Zowe CLI. I also don't think a few conflicting options are that unusual in a CLI.  It's something to watch out for if there are too many though.

Reply:
I'm not really understanding the differing viewpoints 😄 

I like the consistency in options between `zowe jobs submit job ...`  and `zowe zftp submit job ...`